travis/scripts/00-install-jhipster.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -e
+set -ex
```

Comment:
There is a strange error so I enable the bash option « trace for each command after it expands the command and before it executes  it » to see accurately where there is a bug. Advise at https://docs.travis-ci.com/user/installing-dependencies/#Installing-Projects-from-Source. 

**Note:** when I run `$ ./build-samples.sh build ngx-default` or `yarn prettier:format` or on my computer, all work fine. Strange!